_studio/mfx_lib/encode_hw/h264/include/mfx_h264_encode_hw_utils.h

```diff
@@ -1630,32 +1633,31 @@ namespace MfxHwH264Encode
             }
             return MFX_BRC_OK;    
         }
-        mfxU8  GetQp(const BRCFrameParams& par)
+        mfxU8  GetQp(BRCFrameParams& par)
         {
-            mfxBRCFrameParam frame_par={};
-            mfxBRCFrameCtrl  frame_ctrl={};
+            mfxBRCFrameParam &frame_par=par.frameParam;
+            mfxBRCFrameCtrl  &frame_ctrl=par.frameCtrl;
```

Comment:
@ogolikov did you remove all ```const``` qualifiers only to avoid copies for structs above?